File: simple/java/decompose-loop-before.java
void printProperties(List users) {
    for (int i = 0; i < users.size(); i++) {
        String result = null;
        result += users.get(i).getName();
        result += users.get(i).getage();
        System.out.println(result);

        // ...
    }
}

File: simple/php/decompose_loop_before.php
protected function printProperties($users)
{
    for ($i = 0; $i < $users->size(); ++$i)
    {
        $result = NULL;
        $result += $users->get($i)->getName();
        $result += $users->get($i)->getAge();
        echo $result;
    }
}
